src/main/resources/bungee.yml

```diff
@@ -0,0 +1,4 @@
+name: EbeanWrapper
+version: 1.0
```

Comment:
The version should be handled by the semantic-release plugin in the `update-version.sh` script. But I can also add this later.